src/components/Dashboard/widgets/TokenDetailWidget.tsx

```diff
@@ -106,6 +107,13 @@ const MetadataItem = ({ name, value, copyValue }: MetadataItemProps) => (
 const TokenDetail: React.FC = () => {
   const tokenDetail = useSelector(selectCurrentTokenDetail);
 
+  const hasNumberInCollection =
+    tokenDetail.dataAsJson?.arbitraryAsJson?.number_in_collection ||
```

Comment:
I would put tokenDetail.dataAsJson?.arbitraryAsJson? in a separate variable, just to make the code a bit more concise